src/XMakeBuildEngine/Evaluation/ProjectParser.cs

```diff
@@ -186,6 +187,62 @@ private void ParseProjectElement(XmlElementWithLocation element)
             // so we have to set it now
             _project.SetProjectRootElementFromParser(element, _project);
 
+            if (element.HasAttribute(XMakeAttributes.sdk))
+            {
+                // TODO: don't get root of SDKs from the environment, use a built-in or toolset prop
+                // TODO: Don't null ref if MSBuildSDKsPath isn't defined
+
+                var sdksString = element.GetAttribute(XMakeAttributes.sdk);
+
+                var sdks = sdksString.Contains(";") ? sdksString.Split(';') : new[] {sdksString};
+
+                foreach (var sdk in sdks)
+                {
+                    var slashIndex = sdk.LastIndexOf("/", StringComparison.InvariantCulture);
+                    string sdkName = slashIndex > 0 ? sdk.Substring(0, slashIndex) : sdk;
+
+                    // TODO: do something other than just ignore the version
+
+                    if (sdkName.Contains("/"))
+                    {
+                        ProjectErrorUtilities.ThrowInvalidProject(element.GetAttributeLocation(XMakeAttributes.sdk),
+                            "InvalidSdkFormatTooManySlashes");
+                    }
+
+                    // TODO: paths should just be Sdk.props/targets; Sdk-aware imports should do the rest of the path.
+                    var initialImportPath = Path.Combine(Environment.GetEnvironmentVariable("MSBuildSDKsPath"),
+                        sdkName, "Sdk", "Sdk.props");
+                    var finalImportPath = Path.Combine(Environment.GetEnvironmentVariable("MSBuildSDKsPath"),
+                        sdkName, "Sdk", "Sdk.targets");
+
+                    if (File.Exists(initialImportPath))
```

Comment:
I think we need to generate an error if an Sdk is specified that is not found.  However, that should be an error at evaluation, not parse time.  To get that behavior for now, I think we should always add both implicit import elements whether the files exist or not, which will mean that all of our Sdks will need to have both `Sdk.props` and `Sdk.targets`.

When we add support for an `Sdk` attribute on an `Import` and implement the Project-level `Sdk` attribute in terms of Sdk imports, we may be able to change how this works.